subworkflows/local/repeat_analysis.nf

```diff
@@ -34,33 +34,14 @@ workflow REPEAT_ANALYSIS {
 
     BCFTOOLS_SORT_TRGT.out.vcf
         .join(BCFTOOLS_INDEX_TRGT.out.csi)
-        .set{ ch_bcftools_query_in }
-
-    ch_bcftools_query_in
-        .map{[['id':'multisample'],it[1]]}
-        .groupTuple()
-        .set{ vcfs }
-
-    ch_bcftools_query_in
-        .map{[['id':'multisample'],it[2]]}
+        .toList()
+        .filter { it.size() > 1 }
+        .flatMap()
+        .map { meta, bcf, csi -> [ [ id : 'multisample' ], bcf, csi ] }
```

Comment:
Don't you want to use the `case ID` here? Looks like the name of the published merge file will be `multisample.bcf.gz`

Reply:
I have no case ID!  #48.

Reply:
Aha 😄